src/Service/EndTurnService.java

```diff
@@ -0,0 +1,13 @@
+package Service;
+
+import Model.Model;
+import Request.EndTurnRequest;
+import Request.iRequest;
+import Result.EndTurnResult;
+
+public class EndTurnService {
+    Model model = Model.getInstance();
+    public EndTurnResult endTurn(EndTurnRequest request){
+        return model.sendAdvanceTurnCommands(request.getUsername());
```

Comment:
Back to the comment above, the end turn service should call an "endTurn" method, not one that does several things, one of which is sending advance turn commands to everyone. You could split it up here or in the model. I've been doing it in the model, but either way would work.